.smithy.lsp.log

```diff
@@ -0,0 +1,368 @@
+[09:57:59] [09:57:58] Created a temporary folder for file contents /var/folders/nn/63tk2_d96mzf32qwypk1l7km0000gs/T/smithy-lsp14316781329401607384
```

Comment:
was this file added intentionally?

Reply:
Please Ignore this file, added unintentionally.